outputs/alertmanager.go

```diff
@@ -87,12 +90,13 @@ func newAlertmanagerPayload(falcopayload types.FalcoPayload, config *types.Confi
 			}
 			continue
 		}
+		safeLabel := reg.ReplaceAllString(i, "_")
```

Comment:
It breaks a previous behavior, with your regexp, the trailing bracket is replaced by a `_` and not just removed like before.

Reply:
Would a replace all ] } ) with "" work for you here?

Reply:
It would, thanks

Reply:
I have made a change that should resolve this.